cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/block.jelly

```diff
@@ -50,6 +50,7 @@ THE SOFTWARE.
             </j:forEach>
         </d:tag>
     </d:taglib>
+    <f:entry title="Workflow DSL Reference" help="/${it.DSL_REF_URL}"/>
```

Comment:
Anyway I would recommend just using a `f:block` and inline that content here (or use `st:include` if you want it in a separate source file).